skins/oasis/css/core/PageHeader.scss

```diff
@@ -3,40 +3,44 @@
 @import 'skins/shared/mixins/flexbox';
 @import 'layout';
 
-.WikiaPageHeader {
-	margin: 0 10px 10px;
-	position: relative;
-
-	&.separator {
-		border-bottom: 1px solid $color-page-border;
-		padding: 0 0 15px;
+.page-header {
```

Comment:
When introducing this new selector we changes many styles, eg. this rule is applied on H1 https://github.com/Wikia/app/blob/130ef8aac02b1631af89ea1739bc5cb9aef3bcb3/skins/oasis/css/core/responsive-font-sizing.scss#L67-67
Is it intended?